packages/flutter_test/lib/src/goldens.dart

```diff
@@ -53,8 +53,15 @@ abstract class GoldenFileComparator {
   /// Compares the pixels of decoded png [imageBytes] against the golden file
   /// identified by [golden].
   ///
-  /// The returned future completes with a boolean value that indicates whether
-  /// the pixels decoded from [imageBytes] match the golden file's pixels.
+  /// If [isFlaky] is false, the returned future completes with a boolean value
+  /// that indicates whether the pixels decoded from [imageBytes] match the
+  /// golden file's pixels.
+  ///
+  /// If [isFlaky] is true, the comparison always passes. Implementations should
```

Comment:
I'm leaning towards giving it a more self-explanatory name like `alwaysPass` but I don't feel strongly about it.

Reply:
I wanted to give is a name that implies some technical debt. I'm not sure if `alwaysPass` conveys that. But I also don't feel strongly about it.

Reply:
What about `falsePositive`? Or `disable`? I am not a great namer. 🙃

Reply:
`alwaysPassDisabledFalsePositiveFlakyTest` 😛 

Here's my evaluation:

* `isFlaky`:
  * Pros: clear, implies technical debt
  * Cons: could be too specific; perhaps you want to use it for something other than flakiness
* `disable`, `skip`:
  * Pros: clear, implies technical debt; `skip` in particular is already a common name used in `package:test`
  * Cons: makes it sounds as if the golden is not generated at all; it may be surprising that a disabled/skipped check throws an error because the act of screenshotting the widget fails for some reason ("why is it failing? I skipped it!").
* `falsePositive`:
  * Pros: implies technical debt
  * Cons: makes me think 🤔 (which thing is positive? what's false about it?), especially when it can take on both `true` and `false` values, and especially if used in conjunction with `isNot` (i.e. used for false negatives), but `isNot(...falsePositive = false)` makes me think even harder.
* `alwaysPass`:
  * Pros: the most accurate description of the functionality (the check always passes no matter what's in the screenshot)
  * Cons: does not imply technical debt

I think `isFlaky` might be fine, but maybe `skip` or `alwaysPass` are better. `skip` is already a standard name used in the test framework.

Reply:
I think `skip`, especially since it is so widely used in the framework already, indicates the test won't run at all, so this would defy a pretty baked in connotation.
I don't have an opinion otherwise, I am not particularly skilled in crafting names. 🙃